diff --git a/node_modules/toast/src/loader.mjs b/node_modules/toast/src/loader.mjs
index f49c902..685fe24 100644
--- a/node_modules/toast/src/loader.mjs
+++ b/node_modules/toast/src/loader.mjs
@@ -2,6 +2,7 @@ import "./module-aliases.mjs";
 const moduleAliases = {
   react: "preact/compat",
   "react-dom": "preact/compat",
+  "react/jsx-runtime": "preact/jsx-runtime",
 };
 
 export const resolve = (specifier, parentModuleURL, defaultResolve) => {
diff --git a/node_modules/toast/src/module-aliases.mjs b/node_modules/toast/src/module-aliases.mjs
index d550e6a..a236aa6 100644
--- a/node_modules/toast/src/module-aliases.mjs
+++ b/node_modules/toast/src/module-aliases.mjs
@@ -3,5 +3,6 @@ import moduleAlias from "module-alias";
 moduleAlias.addAliases({
   react: "preact/compat",
   "react-dom": "preact/compat",
+  "react/jsx-runtime": "preact/jsx-runtime",
   //   'create-react-class': path.resolve(__dirname, './create-preact-class')
 });
